pkg/apis/crd/v1alpha1/types.go

```diff
@@ -15,6 +15,7 @@
 package v1alpha1
 
 import (
+	_ "github.com/golang/mock/mockgen/model"
```

Comment:
why is this change?

Reply:
Forget to delete this. It is used in `make codegen`. Done.